lib/java/logging-service-telemetry/src/main/scala/org/enso/logging/service/telemetry/ApiMessage.scala

```diff
@@ -0,0 +1,95 @@
+package org.enso.logging.service.telemetry
+
+import com.github.plokhotnyuk.jsoniter_scala.core._
```

Comment:
Ech, here is the Scala usage.
- I assume `jsoniter` is using Scala behind the scene?
- thus there is always going to be a transitive dependency on Scala libraries
- and it makes little sense to ask for rewrite of this class to Java, right?

Reply:
Right. If I understand `jsoniter` correctly, it uses Scala macros for producing the codecs, and for that, you need to invoke Scala compiler. So I have created a minimal Scala source.